README.md

```diff
@@ -13,24 +13,24 @@ This list contains only stage 1 proposals and higher that have not yet been with
-|  | [`Function.prototype.toString` revision][function-to-string]     | Michael Ficarra                                 |
-|  | [`global`][global]                                               | Jordan Harband                                  |
-|  | [Rest/Spread Properties][object-rest-spread]                     | Sebastian Markbage                              |
-|  | [Asynchronous Iteration][async-iteration]                        | Domenic Denicola                                |
-|  | [`import()`][dynamic-import]                                     | Domenic Denicola                                |
-|  | [RegExp Lookbehind Assertions][lookbehind]                       | Daniel Ehrenberg                                |
-|  | [RegExp Unicode Property Escapes][unicode-escapes]               | Brian Terlson, Daniel Ehrenberg, Mathias Bynens |
-|  | [RegExp named capture groups][named-groups]                      | Daniel Ehrenberg, Brian Terlson                 |
-|  | [`s` (`dotAll`) flag for regular expressions][dot-all]           | Mathias Bynens, Brian Terlson                   |
-|  | [Legacy RegExp features in JavaScript][regexp-legacy]            | Mark Miller, Claude Pache                       |
-|  | [`Promise.prototype.finally`][finally]                           | Jordan Harband                                  |
-|  | [BigInt][bigint]                                                 | Daniel Ehrenberg                                |
-|  | [Class Fields][class-fields]                                     | Daniel Ehrenberg, Jeff Morrison                 |
-|  | [Optional catch binding][optional-catch]                         | Michael Ficarra                                 |
-|  | [`import.meta`][import-meta]                                     | Domenic Denicola                                |
-|  | [Private methods and accessors][private-methods]                 | Daniel Ehrenberg, Kevin Gibbons                 |
+| :rocket: | Proposal                                                 | Champion                                        | Tests |
+|--|------------------------------------------------------------------|-------------------------------------------------|-------|
+|  | [`Function.prototype.toString` revision][function-to-string]     | Michael Ficarra                                 | [✅](https://github.com/tc39/test262/issues/1163) |
+|  | [`global`][global]                                               | Jordan Harband                                  | [✅](https://github.com/tc39/test262/issues/765)  |
+|  | [Rest/Spread Properties][object-rest-spread]                     | Sebastian Markbage                              | [✅](https://github.com/tc39/test262/issues/865)  |
+|  | [Asynchronous Iteration][async-iteration]                        | Domenic Denicola                                | [🚧](https://github.com/tc39/test262/issues/923)  |
```

Comment:
1. can we use `:keyword:` syntax here instead of pasting the emoji? i think the emoji might break github's web editor.
1. can we get some kind of legend? I have no idea what the gate or the question mark mean (although the checkmark works)
1. it'd probably be ideal if the checkmarks linked to the relevant test262 PR - we can use the footnote-link style to avoid the lines becoming too long

Reply:
While I'm happy to fix this, I have no idea on what keywords I should use for the respective emojis.

Reply:
`:white_check_mark:` → :white_check_mark:
`:construction:` → :construction: 
`:question:` → :question: